3_javascript/javascript1/week1/javascript.js

```diff
@@ -0,0 +1,104 @@
+ //hello... Hello... This is my first experience in Javascript :)
+
+console.log("******    Age-ify (A future age calculator)    ******");
+let myYearOfBirth = 1991;
+let yearFuture = 2027;
+let age = yearFuture - myYearOfBirth;
+
+console.log("I\'ll be " + age + " years old in " + yearFuture + ".");
+console.log("\n");
+
+
+
+console.log("******    Goodboy-Oldboy (A dog age calculator)    ******");
+let dogYearOfBirth = 2017;
+let dogYearFuture = 2045;
+// We have a conditional statement, so a "BOOLEAN variable" can represent (shouldShowResultInDogYears).
+let shouldShowResultInDogYears = true;
+
+ // According to the information in the txt:(1 human year = 7 dog year)
+let dogAge = (dogYearFuture - dogYearOfBirth)*7;
+let humanAge = dogYearFuture - dogYearOfBirth;
+
+if (shouldShowResultInDogYears) {
+    console.log("His dog will be " + dogAge + " dog years old in " + dogYearFuture + ".");
+}
+
+else {
+ console.log("His dog will be " + humanAge + " human years old in " + dogYearFuture + ".");
+}
+console.log("/* His dog will be 28 human years old in 2045. */");
```

Comment:
Great! again, just replace the `let` keywords with `const` and since you are already stating the solution within the condition, you can delete the hardcoded lines

```
console.log("/* His dog will be 28 human years old in 2045. */");
console.log("\n");
```

Reply:
done!

Reply:
just did not know should deleted this part also or not :
" console.log("\n") ;"

Reply:
yes, you can delete it as well :)